packages/web/src/components/search/SearchBox.js

```diff
@@ -2398,6 +2573,9 @@ const mapStateToProps = (state, props) => ({
 		(state.selectedValues[props.componentId]
 			&& state.selectedValues[props.componentId].value)
 		|| null,
+	selectedImageValue: (state.selectedValues[props.componentId]
+		&& state.selectedValues[props.componentId].meta
+		&& state.selectedValues[props.componentId].meta.imageValue) || null,
```

Comment:
Use `""` (empty string) instead of `null` as fallback value

Reply:
@SavvyShah Unaddressed comment

Reply:
Already addressed.